### ISSUE TITLE
Exception in lavalink Events creates weird issues

### ISSUE THREAD

**Issue by Zenrac** [Devoxin/Lavalink.py#33]
It seems that raising an exception (any exception) in a` lavalink.event` will stuck the event.

How to repro (with v2 or v3 example) : 

```
        elif isinstance(event, lavalink.Events.TrackEndEvent):
            print("TrackEndEvent called")
            raise Exception('This error will block the nexts TrackEndEvent')    
```
You can try yourself to play some songs and see that the TrackEndEvent is stuck

**Comment by Zenrac** [comment]
My guess is that the issue is from : https://github.com/Devoxin/Lavalink.py/blob/master/lavalink/Client.py#L52

**Comment by Devoxin** [comment]
Fixed in 522bc10507a0c7deec5d1e6df8632bb86e5bebcb
